<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乳化液分析</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
   <item>
    <widget class="CardWidget" name="CardWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="SubtitleLabel" name="SubtitleLabel">
          <property name="text">
           <string>数据预览窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ComboBox" name="ComboBox">
          <property name="text">
           <string>选择箱号</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="TableView" name="TableView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="PushButton" name="importRawDataPushButton">
          <property name="text">
           <string>导入并格式化</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/导入.png</normaloff>:/icons/icons/导入.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="importProcessDataPushButton">
          <property name="text">
           <string>导入处理数据</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/导入.png</normaloff>:/icons/icons/导入.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="exportPushButton">
          <property name="text">
           <string>导出</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/导出.png</normaloff>:/icons/icons/导出.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="mergeOnlineTablePushButton">
          <property name="text">
           <string>合并酸轧判定</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/format.png</normaloff>:/icons/icons/format.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="TitleLabel" name="TitleLabel_2">
          <property name="text">
           <string>数据分析</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="2">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="PrimaryPushButton" name="BoxDiagramPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>导入单个酸轧在线判定表</string>
          </property>
          <property name="text">
           <string>乳化液箱线</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="IconWidget" name="IconWidget_8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/箱线图.png</normaloff>:/icons/icons/箱线图.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="IconWidget" name="IconWidget_7">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/跟踪.png</normaloff>:/icons/icons/跟踪.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="PrimaryPushButton" name="IUScatterPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>导入单个酸轧在线判定表</string>
          </property>
          <property name="text">
           <string>浓度与IU散点</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="PrimaryPushButton" name="SpecificTimePushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>导入单个酸轧在线判定表</string>
          </property>
          <property name="text">
           <string>特定时间IU均值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="IconWidget" name="IconWidget_6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc">
            <normaloff>:/icons/icons/散点图.png</normaloff>:/icons/icons/散点图.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="4">
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>47</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../anaconda3/envs/GUI/Lib/site-packages/qtResource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
